extensions/unsplash/src/functions/gridItemSize.ts

```diff
@@ -0,0 +1,15 @@
+import { Grid, getPreferenceValues } from "@raycast/api";
+
+const preferences: any = getPreferenceValues();
+
+export const getGridItemSize = (): Grid.ItemSize => {
+  return preferences.gridItemSize === "small"
+    ? Grid.ItemSize.Small
+    : preferences.gridItemSize === "large"
+    ? Grid.ItemSize.Large
+    : Grid.ItemSize.Medium;
```

Comment:
My extension tells me to move them to separate if statements, that way it'll also look more elaborate.